packages/core/useUrlSearchParams/index.ts

```diff
@@ -21,6 +21,11 @@ export interface UseUrlSearchParamsOptions<T> extends ConfigurableWindow {
    * @default {}
    */
   initialValue?: T
+
+  /**
+   * @default false
+   */
+   onlyRead?: boolean
```

Comment:
```suggestion
  /**
   * Write back to `window.history` automatically
   *
   * @default true
   */
  write?: boolean
```

Reply:
Thanks for review. I have revised it as suggested. The variable is called `write` has been declared below so I changed it to `autoWrite`